lib/archive-utils.rb

```diff
@@ -20,5 +15,4 @@ module Archive
 require 'archive/fixity'
 require 'archive/operating_system'
 require 'archive/tarfile'
-include Archive
```

Comment:
That will work only so long as the down-stream projects have version pins and it's not clear that they do yet.